src/bors/handlers/review.rs

```diff
@@ -0,0 +1,261 @@
+use std::sync::Arc;
+
+use crate::bors::command::Approver;
+use crate::bors::handlers::labels::handle_label_trigger;
+use crate::bors::Comment;
+use crate::bors::RepositoryState;
+use crate::github::GithubUser;
+use crate::github::LabelTrigger;
+use crate::github::PullRequest;
+use crate::permissions::PermissionType;
+use crate::PgDbClient;
+
+/// Approve a pull request.
+/// A pull request can only be approved by a user of sufficient authority.
+pub(super) async fn command_approve(
+    repo_state: Arc<RepositoryState>,
+    db: Arc<PgDbClient>,
+    pr: &PullRequest,
+    author: &GithubUser,
+    approver: &Approver,
+) -> anyhow::Result<()> {
+    tracing::info!("Approving PR {}", pr.number);
+    if insufficient_approve_permission(repo_state.clone(), author) {
+        deny_approve_request(&repo_state, pr, author).await?;
+        return Ok(());
+    };
+    let approver = match approver {
+        Approver::Myself => author.username.clone(),
+        Approver::Specified(approver) => approver.clone(),
+    };
+    tracing::info!("Approving PR {} by {}", pr.number, approver);
+    db.approve(repo_state.repository(), pr.number, approver.as_str())
+        .await?;
+    tracing::info!("Approved PR {}", pr.number);
+    handle_label_trigger(&repo_state, pr.number, LabelTrigger::Approved).await?;
+    tracing::info!("Label trigger handled for PR {}", pr.number);
```

Comment:
This log seems superfluous, such logs are already in `handle_label_trigger`.

Reply:
Ah I forgot to remove those 😅